<html xmlns:py="http://genshi.edgewall.org/"
  xmlns:xi="http://www.w3.org/2001/XInclude">
  
  <py:def function="page_title">Current Value Calculator</py:def>

  <div py:match="content">
  <p py:if="c.error_message" style="border: solid 2px red;">
    There was an error. Details: ${c.error_message}
  </p>
  <form action="${c.ownPath}" method="GET">
		Year: <input name="year" value="${year}" />
		<br />
		<input type="submit" value="Get Current Value" />
	</form>
	<p>
			Value of one pound from ${c.year} in 2002 was: ${c.value}
	</p>
  <p>
  <a href="${c.data_url}">Data Source</a>
  </p>
  </div>

  <xi:include href="layout.html" />
</html>
